week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,7 +1,30 @@
 // Implement the missing functionality according to the task description.
 // Hint: Remember which function you need to call when it's time to alarm.
-
-function setAlarm() { }
+let setTime = document.getElementById("alarmSet");
+let count = document.getElementById("timeRemaining");
+let timer;
+function setAlarm() {
+  clearInterval(timer);
+  let num = setTime.value;
+  timer = setInterval(counter, 1000)
+    function counter() {
+   if (num > 0) num--;
+   num < 10
```

Comment:
in this part of your function you need {} for the if statement. Need to be fixed.